x-pack/platform/plugins/shared/alerting/server/routes/gaps/apis/gap_auto_fill_schedule/logs/get_auto_fill_scheduler_logs_route.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import type { IRouter } from '@kbn/core/server';
+import { schema } from '@kbn/config-schema';
+import type { GapAutoFillSchedulerLogsResponseV1 } from '../../../../../../common/routes/gaps/apis/gap_auto_fill_scheduler';
+import { gapAutoFillSchedulerLogsRequestQuerySchema } from '../../../../../../common/routes/gaps/apis/gap_auto_fill_scheduler';
+import type { ILicenseState } from '../../../../../lib';
+import { verifyAccessAndContext } from '../../../../lib';
+import type { AlertingRequestHandlerContext } from '../../../../../types';
+import { INTERNAL_ALERTING_GAPS_AUTO_FILL_SCHEDULER_API_PATH } from '../../../../../types';
+import { transformRequestV1, transformResponseV1 } from './transforms';
+import { DEFAULT_ALERTING_ROUTE_SECURITY } from '../../../../constants';
+
+export const getAutoFillSchedulerLogsRoute = (
```

Comment:
this should be `findAutoFillSchedulerLogsRoute` instead since you're passing in search parameters.